src/utils/dom.js

```diff
@@ -102,16 +103,16 @@ export function removeChildren(parent) {
 
 /**
  * Injects the provided styles in the HEAD section of the document.
- * @param {!Document} doc The document object.
+ * @param {!Doc} doc The document object.
```

Comment:
I also see `loading-view-test.js` and `dom-test.js` using `injectStyleSheet`. Do they need to be also updated?